packages/dataclass-editor/src/components/dataclass/data/dataclass.ts

```diff
@@ -0,0 +1,61 @@
+export const dataClassFieldNameAttribute = 'name';
```

Comment:
I have not found a satisfying solution for this. This way, it gets defined twice in the same file. This makes it pretty simple to refactor (if you know it is defined twice of course).

Another possibility would be to do this:
```typescript
export const dataClassFieldNameAttribute = 'name';
export const dataClassFieldTypeAttribute = 'type';
export const dataClassFieldCommentAttribute = 'comment';

export interface DataClassField {
  [dataClassFieldNameAttribute]: string;
  [dataClassFieldTypeAttribute]: string;
  [dataClassFieldCommentAttribute]: string;
  modifiers: Array<DataClassFieldModifier>;
  annotations: Array<string>;
  entity?: DataClassFieldEntity;
}
```
This way, it would only be defined once. However, the drawback is that you can no longer reference this attribute using `dataClass.name`. You would need to do `dataClass[dataClassFieldNameAttribute]` whenever you need to get its value.

Since a refactoring of the names of these attributes is very unlikely anyway, I prefer the current implementation.